SimplenoteUITests/UIDs.swift

```diff
@@ -38,7 +38,7 @@ enum UID {
         static let trashNote = "Move to Trash"
         static let restoreNote = "Restore Note"
         static let deleteNote = "Delete Note"
-        static let trashEmptyTrash = "Empty"
+        static let trashEmptyTrash = "Empty trash"
```

Comment:
```suggestion
        static let trashEmptyTrash = "Empty"
```

Can the newer version of Xcode generate the button label differently? 

The code from this branch, running on my laptop (Xcode 12.4), with empty trash button label being set to `Empty trash` fails, because the actual label is still `Empty`. I had to change it back to `Empty` in code to have it working. At the same time, the same code runs on CI (with Xcode 12.5, from what I see) with no fails in this place.

Since I'm now at coworking, I'd postpone downloading 11 Gb of Xcode 12.5 to check my theory about label difference 😅 

Can I suggest keeping the old 'Empty' label, but extending the locator for this button to work with predicate instead of exact match, so that it works for both `Empty` and `Empty trash`?

in `Trash.swft`:
```swift
    class func empty() {
        Trash.open()
        let predicate = NSPredicate(format: "label LIKE '\(UID.Button.trashEmptyTrash)'")
        let emptyTrashButton = app.buttons.element(matching: predicate).firstMatch
        guard emptyTrashButton.isEnabled else { return }

        emptyTrashButton.tap()
        app.alerts.scrollViews.otherElements.buttons[UID.Button.yes].tap()
    }
```

Reply:
On the second thought, feel free to keep it the way it is now, since it works on CI anyway, and I doubt anyone besides me will run these test locally using Xcode 12.4 in nearest few days.

 I'll be creating the PR that solves the instability you mention today, and will include the change there.

In other aspects, good to go from my side :shipit: 
Thank you!

Reply:
Just updated to 12.5, and now the label is `Empty trash` 🧙‍♂️

Reply:
Wow! So it _was_ because of the new Xcode 😱 Go figure...

@pachlava, for future reference, something we could have done during a transition period is adopt the approach from [the Xcode 12.5 WooCommerce PR](https://github.com/woocommerce/woocommerce-ios/pull/4459/files#diff-7c41cb2b963b257aa8753b3c474f818eee259dc7ea8d8eb7653b7031423e2affR69): `if #available(iOS 14.4, *) { ... } else { ... }`

Something like:

```swift
static private(set) lazy var trashEmptyTrash: String = {
  if #available(iOS 14.4, *) {
    "Empty trash"
  } else }
    "Empty
  }
}()
```

Reply:
Did this as a part of #1335, thanks! I agree that this is better than using `*Empty*`. But, it took me some time to understand that asterisk means 'mentioned iOS and higher'. Spent about an hour trying to understand why the execution goes inside `if #available(iOS 14.4, *)` block while running 14.5 😅 Final code I used is:

```swift
static private(set) var trashEmptyTrash: String = {
    if #available(iOS 14.5, *) {
        return "Empty trash"
    } else {
        return "Empty"
    }
}()
```